app-scientific/sagemath/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=sagemath
+PKGSEC=math
+PKGDEP="glibc boost curl gc git gmp isl libatomic-ops libffi libjpeg-turbo \
+        libpng libxml2 mpc mpfr ncurses python-3 zlib openblas ecl gf2x ntl"
+BUILDDEP="openblas ecl gf2x ntl"
+PKGDES="A free open-source mathematics software system"
```

Comment:
A free and open source mathematics software system.

I would even drop the "free and open source" part since it really is not that useful piece of information.